packages/sdk/client/src/echo/index.ts

```diff
@@ -12,8 +12,9 @@ export {
   getTypename,
   Expando,
   type EchoReactiveObject,
-  type ReactiveObject,
   type ObjectMeta,
+  type ReactiveObject,
+  type S, // TODO(burdon): Export as Schema?
```

Comment:
wasn't it suggested that this shouldn't be reexported?

Reply:
we're currently re-exporting it from echo-schema.
do you think we shoudldn't do that?